samples/rtc_example.cpp

```diff
@@ -128,14 +128,12 @@ int main(int argc, char *argv[])
   
   const char *opts[] = {"-std=c++11",
                         "-I/usr/include/linux",
-                        "-I/usr/include/c++/7.3.0",
                         "-I/usr/local/cuda/include",
```

Comment:
This should probably point at the include path in this repo, to make sure it's in line with the includes? Pointing it at the install will not work if the install is in a custom directory, and would also mean you may get headers from a different version, and if your install in this place is 10.2, that'd explain why it fails to run for you (10.2 nvrtc support was... not entirely independent of host headers).

Reply:
> This should probably point at the include path in this repo, to make sure it's in line with the includes?

That sounds like a good idea. Not sure how to do that in a generic/portable way though. Perhaps require that a `LIBCUDACXX_ROOT` environment variable be set to the path of the libcudacxx install?